src/components/Common/Avatar.tsx

```diff
@@ -64,18 +65,25 @@ function Avatar({
       }}
       {...props}
     >
-      <AvatarPrimitive.Image
-        data-slot="avatar-image"
-        src={imageUrl}
-        alt={name}
-        className={cn(
-          "aspect-square size-full object-cover rounded-md",
-          className,
-        )}
-      />
+      {imageUrl && (
```

Comment:
Let's keep AvatarPrimitive.Image, check Radix UI's docs, you should be able to make use of existing props for this.